src/Doctrine/EntityClassGenerator.php

```diff
@@ -59,6 +60,10 @@ public function generateEntityClass(ClassNameDetails $entityClassDetails, bool $
             $useStatements->addUseStatement(class_exists(ApiResource::class) ? ApiResource::class : \ApiPlatform\Core\Annotation\ApiResource::class);
         }
 
+        if ($useUUIDIdentifier) {
```

Comment:
Perhaps `$useUid` instead of `$useUUID` as I would assume uuid or ulid's would both be possible.